clients/js/src/plugins/appData.ts

```diff
@@ -54,15 +54,15 @@ export function appDataInitInfoArgsToBase(
     initPluginAuthority: d.initPluginAuthority
       ? pluginAuthorityToBase(d.initPluginAuthority)
       : null,
-    schema: d.schema ? d.schema : null,
+    schema: d.schema !== undefined && d.schema !== null ? d.schema : null,
```

Comment:
This can be simplified with the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)

Reply:
done